src/Host.cpp

```diff
@@ -3212,17 +3212,24 @@ bool Host::echoWindow(const QString& name, const QString& text)
         return -1;
```

Comment:
That looks suspect - it is not `0` so does that not get converted to a `(bool) true` - when it should be false - OTOH this situation would only apply if `echoWindow` was called when the main profile `TConsole` is not present for the `Host` instance to access it - so the return value is probably the least of the application's problems!

Reply:
True. Still, I'll fix that while I'm at it.